### DISCUSSION TITLE
Version 1.1

### DISCUSSION THREAD

**Discussion by ksqsf** [rimeinn/rime-moran#16] [Announcements]
**Full Changelog**: https://github.com/ksqsf/rime-moran/compare/1.0...1.1

主要更新：
- 新增：詞輔功能、字形標準選擇功能等諸多功能
- 新增：新皮膚
- 大量編碼優化

部分功能可由用戶定製，請參見 moran.defaults.yaml 和 moran_fixed.defaults.yaml 中的說明。

<hr /><em>This discussion was created from the release <a href='https://github.com/ksqsf/rime-moran/releases/tag/1.1'>Version 1.1</a>.</em>

**Comment by kuqquk** [comment]
好用！支持！感谢！（拭泪……）
